lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/collection/IncludeExcludeWidget.py

```diff
@@ -0,0 +1,77 @@
+from ..common.list import StringList
+from ..common.resizable import Resizable
+from ..common.kebabMenu import KebabMenu
+from .ExpressionRulesMenu import ExpressionMenu
```

Comment:
I guess it is too late because I know git is finnicky about these things, but I wish the casing of files was more consistent. Now we have some with initial uppercase and some with initial lowercase.

Reply:
That's a fair point. I can try to change it